### PR TITLE
Add django commands for local development

### PR DESCRIPTION
**Pull request by maurodoglio**
A couple of changes that make local development of the taar api a bit easier.

### CODE REVIEW

File: taar_api/hbase/management/commands/populate_sample_data.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+class Command(BaseCommand):
+    help = 'Populate the HBase table with some sample data'
+
+    def handle(self, *args, **options):
+        conn = Connection(settings.HBASE_HOST)
+        if settings.HBASE_TABLE.encode() not in conn.tables():
+            self.stderr.write(
+                self.style.ERROR('Table "%s" not found.' % settings.HBASE_TABLE))
+        table = conn.table(settings.HBASE_TABLE)
+        data = json.dumps({
```
Comment (Dexterp37) [inline taar_api/hbase/management/commands/populate_sample_data.py]: Each sample in the real table contains much less data, see [here](https://github.com/mozilla/telemetry-batch-view/blob/master/src/test/scala/com/mozilla/telemetry/views/HBaseAddonRecommenderViewTest.scala#L10-L22). Do we care?

File: docker-compose.yml [hunk 2]
```diff
@@ -24,8 +24,11 @@ services:
       - ALLOWED_HOSTS=localhost,127.0.0.1,
       - SECRET_KEY=59114b6a-2858-4caf-8878-482a24ee9542
       - CACHE_URL=redis://redis:6379/1
-      - LOGGING_USE_JSON=0
+      - LOGGING_USE_JSON=1
```
Comment (Dexterp37) [inline docker-compose.yml]: What is this change for?

**Comment by birdsarah** [comment]
This has been open for 6 months. Should either get merged or closed. @mlopatka

**Comment by mlopatka** [comment]
@crankycoder If I'm not mistaken, these changes have been (are currently?) rolled into another refactoring on a branch you started.
Can we abandon this PR in favour of a parallel branch?

**Pull request closed without merging**